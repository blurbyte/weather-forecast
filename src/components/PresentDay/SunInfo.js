import React from 'react';

import Row from '../Row';
import InfoText from './InfoText';
import Label from './Label';

const SunInfo = () => (
  <Row>
    <InfoText>
      <Label>Sunrise</Label> 5:50
    </InfoText>
    <InfoText>
      <Label>Sunset</Label> 20:45
    </InfoText>
  </Row>
);

export default SunInfo;
