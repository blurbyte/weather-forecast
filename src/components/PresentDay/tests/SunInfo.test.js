import React from 'react';
import renderer from 'react-test-renderer';

import SunInfo from '../SunInfo';

test('renders correctly', () => {
  const component = renderer.create(<SunInfo />);
  expect(component.toJSON()).toMatchSnapshot();
});
