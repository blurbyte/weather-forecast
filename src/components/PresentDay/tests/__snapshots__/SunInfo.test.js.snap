// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa DYOsF"
>
  <p
    className="sc-htpNat hwCFSM sc-bwzfXH fBSZGv"
  >
    <span
      className="sc-bxivhb iBbKEM"
    >
      Sunrise
    </span>
     5:50
  </p>
  <p
    className="sc-htpNat hwCFSM sc-bwzfXH fBSZGv"
  >
    <span
      className="sc-bxivhb iBbKEM"
    >
      Sunset
    </span>
     20:45
  </p>
</div>
`;
